helm/auto-deployment/configuration.md

```diff
@@ -0,0 +1,444 @@
+       <b>Description</b>
+      </td>
+    </tr>
+    <tr>
+      <td>
+       clusterProvider
+      </td>
+      <td>
+       The type of kubernetes cluster provider you are using. In this case the clusterProvider value is <i>eks</i>
+      </td>
+    </tr>
+    <tr>
+      <td>
+       persistentVolume.eks.efsFileSystemId
+      </td>
+      <td>
+       The <i>File system ID</i> of your EFS file system.
+      </td>
+    </tr>
+</table>
+<br/>
+
+3. AKS
+
+```console
+clusterProvider: aks
+    
+persistentVolume:
+  aks:
+    fileShareName: <file_share_name>
+    # base64 string
+    azureStorageAccountName: <base64_azurestorageaccountname>
+    # base64 string
+    azureStorageAccountKey: <base64_azurestorageaccountkey>
+```
+
+<br/>
+<table>
+    <tr>
+      <td>
+       <b>Name</b>
+      </td>
+      <td>
+       <b>Description</b>
+      </td>
+    </tr>
+    <tr>
+      <td>
+       clusterProvider
+      </td>
+      <td>
+       The type of kubernetes cluster provider you are using. In this case the clusterProvider value is <i>aks</i>
+      </td>
+    </tr>
+    <tr>
+      <td>
+       persistentVolume.aks.fileShareName
+      </td>
+      <td>
+       The <i>File share name</i> of your File share instance.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       persistentVolume.aks.azureStorageAccountName
+      </td>
+      <td>
+       The <i>base64 encoded storage account name</i> of the File share instance in your storage account.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       persistentVolume.aks.azureStorageAccountKey
+      </td>
+      <td>
+       The <i>base64 encoded storage account key</i> of the File share instance in your storage account.
+      </td>
+    </tr>
+</table>
+<br/>
+
+> **NOTE:** The Azure storage account credentials will be maintained in a secret named `bold-azure-secret`
+
+4. On-Premise
+  
+```console
+clusterProvider: onpremise
+    
+persistentVolume:
+  onpremise:
+    hostPath: /run/desktop/mnt/host/<local_directory>
+```
+
+<br/>
+<table>
+    <tr>
+      <td>
+       <b>Name</b>
+      </td>
+      <td>
+       <b>Description</b>
+      </td>
+    </tr>
+    <tr>
+      <td>
+       clusterProvider
+      </td>
+      <td>
+       The type of kubernetes cluster provider you are using. In this case the clusterProvider value is <i>onpremise</i>
+      </td>
+    </tr>
+    <tr>
+      <td>
+       persistentVolume.onpremise.hostPath
+      </td>
+      <td>
+       The shared folder path in your host machine.<br/><br/>
+       Ex: Create a folder in your machine to store the shared folders for application usage.
+       <i>D://app/shared</i><br/><br/>
+       The corresponding value for the above shared folder will be,
+       <i>/run/desktop/mnt/host/d/app/shared</i>
+      </td>
+    </tr>
+</table>
+<br/>
+
+## Load Balancing
+
+### Ingress-Nginx
+
+By default Nginx is used as reverse proxy for Bold BI. To configure load balancing in Bold BI take a look at the below snipet. 

```

Comment:
By default, Nginx is used as a reverse proxy for Bold BI. To configure load balancing in Bold BI, take a look at the following code sample.

Reply:
concern addressed